src/pages/komodo-defi-framework/api/common_structures/index.mdx

```diff
@@ -499,3 +499,84 @@ The `PagingOptions` object includes options of page selection to consult when lo
+| events    | list of strings | A list of events to track during the connection. For EIP155, refer to [https://docs.reown.com/advanced/providers/ethereum#events](https://docs.reown.com/advanced/providers/ethereum#events). For Cosmos, refer to [https://tutorials.cosmos.network/academy/2-cosmos-concepts/10-events.html](https://tutorials.cosmos.network/academy/2-cosmos-concepts/10-events.html)                |
+
+<CollapsibleSection expandedText="Hide Examples" collapsedText="Show Examples">
+  #### EIP155 Example
+
+  ```json
+  {
+  	"chains": [
+  		"eip155:1",      // ETH
+  		"eip155:56",     // BNB
+  		"eip155:137",    // POLYGON
+  		"eip155:43114"   // AVAX
+  	],
+  	"methods": [
+  		"eth_sendTransaction",
+  		"eth_signTransaction",
+  		"personal_sign"
+  	],
+  	"events": [
+  		"accountsChanged",
+  		"chainChanged"
+  	]
+  }
+  ```
+
+  #### Cosmos Example
+
+  ```json
+  {
+  	"chains": [
+  		"cosmos:cosmoshub-4",
+  		"cosmos:osmosis-1",
+  		"cosmos:irishub-1"
+  	],
+  	"methods": [
+  	  	"cosmos_signDirect",
+  		  "cosmos_signAmino",
+  		  "cosmos_getAccounts"
+  	],
+  	"events": []
+  }
+  ```
+</CollapsibleSection>
+
+### WcSession
+
+The `WcSession` object contains details of active WalletConnect sessions.
+
+| Parameter        | Type    | Description                                                                                                                                                                                                                                                                |
+| ---------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| topic            | string  | The session topic hex string, used to identify the session for communications with the external wallet/dapp.                                                                                                                                                               |
+| metadata         | object  | Arbitrary data returned from the external wallet/dapp on connection.                                                                                                                                                                                                       |
+| peer\_pubkey     | string  | TBA                                                                                                                                                                                                                                                                        |
```

Comment:
@borngraced please assist with description here

Reply:
Can we remove this field @smk762 this should be kept internally only.